doc/source/users_guide/running-special-cases/Running-with-anomaly-forcing.rst

```diff
@@ -5,41 +5,69 @@
+        <var>rsds  Faxa_swdn_af</var>
+        <var>tas   Sa_tbot_af</var>
+        <var>uas   Sa_u_af</var>
+        <var>vas   Sa_v_af</var>
+     </datavars>
+     ...
+  </stream_info>
 
-  anomaly\_forcing = 'Anomaly.Forcing.Temperature'
+To use alternative data, add a ``user_nl_datm_streams`` namelist file to your case with contents like so:
 
-which will only adjust the temperature (TBOT).
+::
 
-After the namelist has been created, the run directory will be populated with files such as these:
+  Anomaly.Forcing.cmip6.ssp585:datafiles = /path/to/your/datafile
+  Anomaly.Forcing.cmip6.ssp585:meshfile = /path/to/meshfile/for/your/datafile
 
-  datm.streams.txt.Anomaly.Forcing.Temperature
+  ! List of Data types to use
+  ! Remove the variables you do NOT want to include in the anomaly forcing:
+  !     pr is precipitation
+  !     tas is temperature
+  !     huss is humidity
+  !     uas and vas are U and V winds
+  !     rsds is solare
+  !     rlds is LW down
+  Anomaly.Forcing.cmip6.ssp585:datavars = pr    Faxa_prec_af, \
+                                          tas   Sa_tbot_af, \
+                                          ps    Sa_pbot_af, \
+                                          huss  Sa_shum_af, \
+                                          uas   Sa_u_af, \
+                                          vas   Sa_v_af, \
+                                          rsds  Faxa_swdn_af, \
+                                          rlds  Faxa_lwdn_af
 
-which will contain the location of the default anomaly forcing datasets.  To use alternative data, copy these files to the case directory with the 'user\_' prefix, and modify the 'user\_*' files accordingly, e.g.:
+To instead disable anomaly forcing in ``ISSP`` compsets, the following can be added to the ``user_nl_datm`` file:
 
-  user\_datm.streams.txt.Anomaly.Forcing.Temperature
+::
 
-    For example, one could use the user\_datm.streams.txt.Anomaly.Forcing.* files to point to these SSP-derived anomaly forcing datasets:
+  anomaly_forcing = 'none'
 
-    /glade/p/cesmdata/cseg/inputdata/atm/datm7/anomaly\_forcing/CMIP6-SSP3-7.0
+Note that other inputs are also set automatically for ``ISSP`` compsets, including CO2 (``co2tseries``), ozone (``preso3``), N deposition (``presndep``), and aerosols (``presaero``).
 
-    af.huss.cesm2.SSP3-7.0.2015-2100\_c20200329.nc
-    af.pr.cesm2.SSP3-7.0.2015-2100\_c20200329.nc
-    af.ps.cesm2.SSP3-7.0.2015-2100\_c20200329.nc
-    af.rlds.cesm2.SSP3-7.0.2015-2100\_c20200329.nc
-    af.rsds.cesm2.SSP3-7.0.2015-2100\_c20200329.nc
-    af.tas.cesm2.SSP3-7.0.2015-2100\_c20200329.nc
+For single-point simulations, the global anomaly forcing files can be used, but the map should be nearest-neighbor interpolated by putting the following in ``user_nl_datm_streams``:
 
-Users may wish to also update files such as the landuse\_timeseries and aerosol and Ndepostion files to correspond to the appropriate SSP.
+TODO: IS THIS ACTUALLY NEEDED?
```

Comment:
Good question.

It could be worth trying it without it to make sure it is required for all the fields. Setting it for some fields was required in the past, but may not be anymore.